### PR TITLE
Static proptypes

### PR DESCRIPTION
**Pull request by lalli-flores**

### CODE REVIEW

**Comment by akoushke** [comment]
what is the point of making this `static`?

File: package.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
```
Comment (adamweeks) [inline package.json]: This is required to do static props?

File: src/components/WebexAvatar/WebexAvatar.js [hunk 2]
```diff
@@ -30,5 +30,3 @@ export default class WebexAvatar extends PureComponent {
     return <Avatar src={avatar} title={displayName} type={status} alt={displayName} />;
   }
 }
-
-WebexAvatar.propTypes = propTypes;
```
Comment (adamweeks) [inline src/components/WebexAvatar/WebexAvatar.js]: I like that it removes this strange looking code.

File: package.json (hunk 1 above)
Comment (lalli-flores) [reply package.json]: Yes, `static` is considered experimental since it's still a proposal

**Comment by lalli-flores** [comment]
> what is the point of making this `static`?

It's a cleaner way of attaching a property to an object

**Pull request merged**